bindata/assets/alerts/client-go-requests.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: client-go
+  namespace: openshift-kube-apiserver
+spec:
+  groups:
+    - name: control-plane-client-go
+      rules:
+        - record: client_go_request_duration:method_get:histogram_quantile99
+          expr: |
+            histogram_quantile(0.99,
+              sum(rate(
+                label_replace(rest_client_request_duration_seconds_bucket{verb="GET"},
+                  "url_host","$1$2","url","https?://([^:^/]*)(:[0-9]+)?.*?")[5m:30s]
+                )
+              ) by (url_host,job)
+            )
+        - record: client_go_request_total:network_errors:rate5m
```

Comment:
```suggestion
        - record: cluster:rest_client_requests:rate
```

Reply:
this is about error rate, better `cluster:rest_client_requests_errors:rate`

Reply:
sorry, it was a question, I clicked too fast

Reply:
added the one that you suggested